Sources/Danger/GitHubDSL.swift

```diff
@@ -297,7 +297,7 @@ extension GitHub {
         enum CodingKeys: String, CodingKey {
             case body
             case commitId = "commit_id"
-            case id // swiftlint:disable:this identifier_name
+            case id
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitHubDSL.swift;
  Line: 300;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)